tests/test_utils.py

```diff
@@ -288,6 +467,51 @@ def test_sub_pixel_read_interpolation_modes():
         assert output.shape == out_size
 
 
+def test_sub_pixel_read_incorrect_read_func_return():
+    bounds = (0, 0, 8, 8)
```

Comment:
docstring?

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/8153082e029f43240b42ac8c444182696d078208